recipes/prgenv-gnu/24.11/a100/extra/reframe.yaml

```diff
@@ -8,4 +8,5 @@ default:
   cc: mpicc
   cxx: mpic++
   ftn: mpifort
+  # TODO: Update this and other reframe activations?
   activation: uenv view develop
```

Comment:
Is there someone who knows if these activations still work with the new uenv? Does reframe need changes to support loading a view at `uenv start`-time?

@ekouts you might know best what's supported/required?

Reply:
Reframe has support for loading views at start time.
There is a `view` parameter that can be configured in the `extra/reframe.yaml` file that specifies which view(s) to load.

Reply:
see the cp2k recipe for example:
https://github.com/eth-cscs/alps-uenv/blob/main/recipes/cp2k/2024.3/gh200/extra/reframe.yaml

Reply:
Thanks. Updated (all the prgenv-gnu recipes). If I understand correctly, these recipes would need to be rebuilt and deployed with a new version to be able to use the updated reframe.yaml files, yes?

From my perspective this can wait for the next version bump of the prgenv-gnu uenvs, though it wouldn't hurt to have a prgenv-gnu which works with the new uenv. Feels a bit strange to bump it to v2 with a non-user-facing change, but maybe it doesn't matter. Preferences?